app/views/seller_homepage/seller_orders.html.erb

```diff
@@ -30,14 +30,11 @@
         <span> Contact No. <%=item.useraddress.phonenumber%> </span>
       </td>
       <td>
-        <% if item.product.product_status.present? %>
-          <%=item.product.product_status.status%>
-        <% else %>
-          Pending...  
+        <% if item.product_status.present? %>
+        Current Status <h5> <%=item.product_status.status%> </h5>
```

Comment:
if you are inside the block so use proper indent. here also incorrect indent

Reply:
ok I will correct this